packages/flutter_driver/test/flutter_driver_test.dart

```diff
@@ -35,8 +35,8 @@ void main() {
       when(mockVM.isolates).thenReturn([mockIsolate]);
       when(mockIsolate.loadRunnable()).thenReturn(mockIsolate);
       when(mockIsolate.invokeExtension(any, any))
-          .thenReturn(new Future.value({'status': 'ok'}));
-      vmServiceConnectFunction = (_) => new Future.value(mockClient);
+          .thenReturn(new Future<Map<String, dynamic>>.value(<String, String>{'status': 'ok'}));
```

Comment:
Yeah, tests are always a tricky case.

I was referring more generally to the code that has futures and streams that pass `Map<String, dynamic>s`. It felt like I was adding `<String, dynamic>` annotations to a lot of things, and in many cases it was very unclear what exactly I should be adding; I'm sure I got some wrong. (Bug https://github.com/dart-lang/sdk/issues/25989 didn't help.) Maybe I misunderstood what the code was doing. I would have felt better if all the code that involved "dynamic" had been synchronous and in one place, generating strongly-typed data structures. I don't think we should ever have a Future or Map that passes a `Map<String, dynamic>`, it's too bug-prone (it basically entirely side-steps the static checker).